pkg/api/rest/types.go

```diff
@@ -128,7 +128,31 @@ func (nodeStrategy) Validate(obj runtime.Object) errors.ValidationErrorList {
 	return validation.ValidateMinion(node)
 }
 
-// namespaceStrategy implements behavior for nodes
+// eventStrategy implements behavior for events
+type eventStrategy struct {
+	runtime.ObjectTyper
+	api.NameGenerator
+}
+
+// Events is the default logic that applies when creating and updating event objects.
+var Events RESTCreateStrategy = eventStrategy{api.Scheme, api.SimpleNameGenerator}
```

Comment:
Do we need to make a second strategy to get this to work for both namespaced and non-namespaced events? (I have not finished grokking these strategies.)

@smarterclayton